{
	"id": "oval:org.debian:def:113352790905718278420289788693013584174",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2023-25399 scipy",
		"affected": {
			"family": "unix",
			"platform": "Debian GNU/Linux 11",
			"product": "scipy"
		},
		"reference": [
			{
				"source": "CVE",
				"ref_id": "CVE-2023-25399",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-25399"
			}
		],
		"description": "A refcounting issue which leads to potential memory leak was discovered in scipy commit 8627df31ab in Py_FindObjects() function. Note: This is disputed as a bug and not a vulnerability. SciPy is not designed to be exposed to untrusted users or data directly.",
		"debian": {}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterias": [
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.debian.oval:tst:2",
								"comment": "all architecture"
							},
							{
								"test_ref": "oval:org.debian.oval:tst:27238",
								"comment": "scipy DPKG is earlier than 0"
							}
						]
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.debian.oval:tst:1",
				"comment": "Debian 11 is installed"
			}
		]
	}
}
